src/pages/requests/BroadcastRequest.tsx

```diff
@@ -65,7 +65,11 @@ export const BroadcastRequest = (props: BroadcastRequestProps) => {
     setIsProcessing(true);
     await sleep(25);
 
-    const { txid, message } = await broadcastWithGorillaPool(request.rawtx);
+    let rawtx = request.rawtx;
+    if (request.fund) {
+      rawtx = await fundRawTx(rawtx);
```

Comment:
`fundRawTx` takes a password as a 2nd argument. We need to collect it and also show the outputs, if possible, or at least how much bsv is being sent. I have `satsIn`, `satsOut`, and `fee` variables in the `fundRawTx` function we can return along with the rawtx, if that would be useful.